### ISSUE TITLE
[FR] Add xHTTP

### ISSUE THREAD

**Issue by misagh72** [hiddify/hiddify-app#1508]
### Feature description

HI please support xHTTP

**Comment by poshl9k** [comment]
absolutly for it!

**Comment by zuiwuchang** [comment]
I also want xhttp, thanks.

**Comment by allanchan339** [comment]
Yes just realize that in manager we have xhttp where the description said Hiddify and Xray client is supported. But in fact the Hiddify app is not supported.

**Comment by Haargrad** [comment]
please add xhttp support

**Comment by mytecor** [comment]
xray core update required

**Comment by goodboy34-tech** [comment]
**Make this pls**

**Comment by imoonkin** [comment]
Both singbox and clashMeta implement outbound traffic themselves. They won't implement xhttp, said on their issue section. I think the simplest solution is to add a third core (xray).

**Comment by horanchikk** [comment]
up

**Comment by Runinterface** [comment]
UP! Make it pls.

**Comment by thunderGoose3000** [comment]
Up

**Comment by sliine** [comment]
Up

**Comment by megam19** [comment]
Up

**Comment by Bebrik-Podberezovik** [comment]
up. So strange that hiddify manager with xray core support xhttp, but hiddify app dont
